ios/RNCAsyncStorage.m

```diff
@@ -34,6 +34,26 @@
   }
 }
 
+static BOOL RCTAsyncStorageSetExcludedFromBackup(NSString* path, NSNumber* isExcluded)
+{
+    NSFileManager *fileManager = [[NSFileManager alloc] init];
+    
+    BOOL isDir;
+    BOOL exists = [fileManager fileExistsAtPath:path isDirectory:&isDir];
+    BOOL success = false;
+    
+    if (isDir && exists) {
```

Comment:
Why does it have to be a directory?

Reply:
It does not have to be a directory, actually. It's just a preference to exclude whole dir, instead a file. Any possible drawbacks from that?

Reply:
Apple encourages using a directory in some docs, so if it's reasonable to assume async storage might add some more files in there, I totally support a directory.

If this is a breaking change from previous versions, then I would recommend not introducing a new directory until you need to.

Reply:
That makes sense. I was just wondering whether this particular function should fail because the path is not a directory.